### DISCUSSION TITLE
How to set specific Environment path variable in Visual studio using premake?

### DISCUSSION THREAD

**Discussion by SudharsanZen** [premake/premake-core#1637] [Q&A] [answered]
I want to set the Environment path variables which has the location of the .dll files. I tried looking the documentation but I couldn't find it. Can some one please say how do i set those values?
![VSENV](https://user-images.githubusercontent.com/21234212/120345630-45a4ed00-c318-11eb-8908-e6c8fa73ce0c.PNG)

**Comment by samsinsane** [comment, marked as answer]
Have you tried [`debugenvs`](https://premake.github.io/docs/debugenvs)?

**Reply by SudharsanZen** [reply]
oh! thank you very much, that's exactly what I've been looking for. I should've spent more time on the docs, my bad.
